dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1446,10 +1446,28 @@ a.download-video {
   #answerdest {
     width: 240px;
   }
+  .answerlist {
+    h4 {
+      position: relative;
+      top: 25%;
+      -webkit-transform: translateY(-50%);
+      -ms-transform: translateY(-50%);
+      transform: translateY(-50%);
```

Comment:
i suspect scss has a mixin that will generate the browser-specific variations, so you don't have to manually type them all.